default.nix

```diff
@@ -197,6 +197,9 @@ rec {
   rswitch = import sources.rswitch.outPath {
     pkgs = pkgs-poetry;
   };
+  
+  getRancherCreds = (import sources.conformity-tooling { inherit pkgs;}).getrancherCreds;
+  checkMetrics = (import sources.conformity-tooling {inherit pkgs; }).checkmetrics;
```

Comment:
hyphen and lowercase to stick to nixpkgs naming convention : https://nixos.org/manual/nixpkgs/stable/#sec-package-naming